Sources/Paywalls/Components/PaywallPackageComponent.swift

```diff
@@ -35,6 +35,18 @@ public extension PaywallComponent {
             self.isSelectedByDefault = isSelectedByDefault
             self.stack = stack
         }
+
+    }
+
+}
+
+extension PaywallComponent.PackageComponent {
+
+    enum CodingKeys: String, CodingKey {
+        case type
+        case packageID = "packageId"
+        case isSelectedByDefault = "isDefaultSelected"
```

Comment:
Hmm strange these are using camelCase instead of snake_case? Same below

Reply:
Codables are a bit weird here 😅 

Keys will get deserialized from snake to camel automatically... however, `packageID` would get deserialized as `package_i_d` because of the `ID` both being capitals 🫠 

Some of the other codables are values that we _actualy_ are expecting to be snake case as well but `Codable` only auto converts for keys (not values)

Reply:
That's.... confusing 🥴. But yeah makes sense. Thanks for explaining!

Reply:
It honestly is 😬 Maybe I should document this is all the place? Because it will be more than just me supporting this and I don't want to make anybody else think about Swift quirks 😛

Reply:
Maybe a quick comment could help yeah. Having said that, I think as long as the decoding behavior is well tested, this should be caught with tests